arm/Microsoft.ServiceBus/namespaces/networkRuleSets/deploy.bicep

```diff
@@ -0,0 +1,50 @@
+@description('Conditional. The name of the parent Service Bus Namespace for the Service Bus Network Rule Set. Required if the template is used in a standalone deployment.')
+@minLength(6)
+@maxLength(50)
+param namespaceName string
+
+@description('Required. The default is the only valid ruleset.')
+param name string = 'default'
+
+@description('Optional. Enable telemetry via the Customer Usage Attribution ID (GUID).')
+param enableDefaultTelemetry bool = true
+
+@description('Required. Configure networking options for Premium SKU Service Bus, ipRules and virtualNetworkRules are not required when using dedicated modules.')
+param networkRuleSet object = {}
```

Comment:
Please use separated parameters for child module properties and not the whole object to allow input validation